src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -458,11 +458,7 @@ describe('StoryPromo Container', () => {
     describe('Recommendation Promo', () => {
```

Comment:
We can remove this test too since we don't use StoryPromo & we have recreated this test in CpsRecommendations